project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,217 @@
+# Waypoint Project Template Example
+
+This is an example of how to create and use project templating in HCP Waypoint.
+
+Pre-Requisites
+
+AWS Account with Admin Permissions
+
+Terraform Enterprise Account with No Code Module Enabled.
+
+## Foundational Infrastructure Setup
+
+### Terraform Cloud Workspaces
+
+1. (CLI-driven only) Create terraform cloud [workspaces](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/creating) in the TFC User Interface based on the following terraform configurations provided in this example:
+   `terraform-aws-example-microservice-infra`
+   `terraform-aws-example-network`
+   ![img_9.png](../readme-images/create_tfc_workspace_ss.png)
+
+
+2. Go to Settings for the created TFC workspaces, Under `Remote State Sharing` allow `Share with all workspaces in this organization`.
+   ![img_10.png](../readme-images/tfc_remote_sharing_ss.png)
+
+
+3. Create a [global variable set](https://developer.hashicorp.com/terraform/tutorials/cloud-get-started/cloud-workspace-create#create-the-workspace) for your AWS Credentials.
+
+### AWS Steps with Terraform
+
+Create the baseline infrastructure with the following steps in your CLI.
+
+1. Navigate to the `terraform-aws-example-network` directory found in this example and run `terraform init` and the `terraform apply` command.
+
+2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example and run `terraform init` and the  `terraform apply` command.
```

Comment:
I think it would be useful to have the `terraform plan` listed here too
```suggestion
2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example and run `terraform init`, `terraform plan`, and  `terraform apply`.
```

Reply:
`terraform apply` actually does a `terraform plan` and before applying the changes. It will prompt the user to approve the planned changes before performing the actual apply so the user can review the changes and will let the user know if it detects any drift . So I think we're ok here 😄